distribution/docs/src/main/resources/_contents/_catalog-frameworks/global-attribute-validators-definition-contents.adoc

```diff
@@ -55,7 +55,7 @@ The `validator` key must have a value of one of the following:
  - `futuredate`
  * `arguments`: (0) [NO ARGUMENTS]
  - `range`
- ** (2) [number (decimal or integer): inclusive lower bound, number (decimal or integer): inclusive upper bound]
+ ** (2) [number (decimal or integer): inclusive lower bound, number (decimal or integer): inclusive upper bound, lower bound must be greater than or equal to zero and the upper bound must be greater than or equal to the lower bound]
```

Comment:
Possibly no need to change this file at all.  Seems preconditions aren't covered here anyway